src/Compilers/CSharp/Test/Semantic/Semantics/InterpolationTests.cs

```diff
@@ -17866,5 +17866,24 @@ public void InterpolatedStringInForeach_HasErrors()
                   NextVariables(0)
                 """);
         }
+
+        [ConditionalFact(typeof(CoreClrOnly)), WorkItem("https://github.com/dotnet/roslyn/issues/68834")]
+        public void ParensInBinaryConcat()
```

Comment:
I think I need to understand why `ParenthesizedAdditiveExpression_01` didn't cover this scenario.

Reply:
Also, please add this as another theory parameter on that test, not a new one down here.

Reply:
`DefaultInterpolatedStringHandler` is a ref struct. The existing tests are different because don't use a ref struct handler, so they do a fast exist from `GetValEscape` and never reach `GetInterpolatedStringHandlerConversionEscapeScope`/`GetInterpolatedStringHandlerArgumentsForEscape`

Reply:
Thanks for the explanation.